{
  "uid" : "27c2c12f61744d9",
  "name" : "Verify that links in the sections lead to the correct pages after clicking",
  "fullName" : "tests.used_resources_page_test.TestUsedResourcesPageLinks#test_ur_03_02_verify_links_lead_to_the_correct_pages",
  "historyId" : "8c05a0537ad7d500848fd1ce3276ea0e",
  "time" : {
    "start" : 1728781377925,
    "stop" : 1728781377925,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5613efe4f02a <unknown>\n#1 0x5613efb355e0 <unknown>\n#2 0x5613efb2d7eb <unknown>\n#3 0x5613efb1da09 <unknown>\n#4 0x5613efb1f744 <unknown>\n#5 0x5613efb1dcbd <unknown>\n#6 0x5613efb1d52f <unknown>\n#7 0x5613efb1d422 <unknown>\n#8 0x5613efb1b493 <unknown>\n#9 0x5613efb1bada <unknown>\n#10 0x5613efb37f41 <unknown>\n#11 0x5613efbc98c5 <unknown>\n#12 0x5613efba9b22 <unknown>\n#13 0x5613efbc8d7d <unknown>\n#14 0x5613efba98c3 <unknown>\n#15 0x5613efb776b3 <unknown>\n#16 0x5613efb7868e <unknown>\n#17 0x5613efe19a2b <unknown>\n#18 0x5613efe1d9b1 <unknown>\n#19 0x5613efe06225 <unknown>\n#20 0x5613efe1e532 <unknown>\n#21 0x5613efdeb38f <unknown>\n#22 0x5613efe3df28 <unknown>\n#23 0x5613efe3e0f3 <unknown>\n#24 0x5613efe4de7c <unknown>\n#25 0x7f135e294ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"fd4ec0e2a9fcca331b6307bd22e1dad3\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe0c0e56f30>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5613efe3e0f3 \\\\u003Cunknown>\\\\n#24 0x5613efe4de7c \\\\u003Cunknown>\\\\n#25 0x7f135e294ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5613efe4f02a <unknown>\nE       #1 0x5613efb355e0 <unknown>\nE       #2 0x5613efb2d7eb <unknown>\nE       #3 0x5613efb1da09 <unknown>\nE       #4 0x5613efb1f744 <unknown>\nE       #5 0x5613efb1dcbd <unknown>\nE       #6 0x5613efb1d52f <unknown>\nE       #7 0x5613efb1d422 <unknown>\nE       #8 0x5613efb1b493 <unknown>\nE       #9 0x5613efb1bada <unknown>\nE       #10 0x5613efb37f41 <unknown>\nE       #11 0x5613efbc98c5 <unknown>\nE       #12 0x5613efba9b22 <unknown>\nE       #13 0x5613efbc8d7d <unknown>\nE       #14 0x5613efba98c3 <unknown>\nE       #15 0x5613efb776b3 <unknown>\nE       #16 0x5613efb7868e <unknown>\nE       #17 0x5613efe19a2b <unknown>\nE       #18 0x5613efe1d9b1 <unknown>\nE       #19 0x5613efe06225 <unknown>\nE       #20 0x5613efe1e532 <unknown>\nE       #21 0x5613efdeb38f <unknown>\nE       #22 0x5613efe3df28 <unknown>\nE       #23 0x5613efe3e0f3 <unknown>\nE       #24 0x5613efe4de7c <unknown>\nE       #25 0x7f135e294ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728781378701,
      "stop" : 1728781379300,
      "duration" : 599
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5613efe4f02a <unknown>\n#1 0x5613efb355e0 <unknown>\n#2 0x5613efb2d7eb <unknown>\n#3 0x5613efb1da09 <unknown>\n#4 0x5613efb1f744 <unknown>\n#5 0x5613efb1dcbd <unknown>\n#6 0x5613efb1d52f <unknown>\n#7 0x5613efb1d422 <unknown>\n#8 0x5613efb1b493 <unknown>\n#9 0x5613efb1bada <unknown>\n#10 0x5613efb37f41 <unknown>\n#11 0x5613efbc98c5 <unknown>\n#12 0x5613efba9b22 <unknown>\n#13 0x5613efbc8d7d <unknown>\n#14 0x5613efba98c3 <unknown>\n#15 0x5613efb776b3 <unknown>\n#16 0x5613efb7868e <unknown>\n#17 0x5613efe19a2b <unknown>\n#18 0x5613efe1d9b1 <unknown>\n#19 0x5613efe06225 <unknown>\n#20 0x5613efe1e532 <unknown>\n#21 0x5613efdeb38f <unknown>\n#22 0x5613efe3df28 <unknown>\n#23 0x5613efe3e0f3 <unknown>\n#24 0x5613efe4de7c <unknown>\n#25 0x7f135e294ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728781378702,
        "stop" : 1728781379299,
        "duration" : 597
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5613efe4f02a <unknown>\n#1 0x5613efb355e0 <unknown>\n#2 0x5613efb2d7eb <unknown>\n#3 0x5613efb1da09 <unknown>\n#4 0x5613efb1f744 <unknown>\n#5 0x5613efb1dcbd <unknown>\n#6 0x5613efb1d52f <unknown>\n#7 0x5613efb1d422 <unknown>\n#8 0x5613efb1b493 <unknown>\n#9 0x5613efb1bada <unknown>\n#10 0x5613efb37f41 <unknown>\n#11 0x5613efbc98c5 <unknown>\n#12 0x5613efba9b22 <unknown>\n#13 0x5613efbc8d7d <unknown>\n#14 0x5613efba98c3 <unknown>\n#15 0x5613efb776b3 <unknown>\n#16 0x5613efb7868e <unknown>\n#17 0x5613efe19a2b <unknown>\n#18 0x5613efe1d9b1 <unknown>\n#19 0x5613efe06225 <unknown>\n#20 0x5613efe1e532 <unknown>\n#21 0x5613efdeb38f <unknown>\n#22 0x5613efe3df28 <unknown>\n#23 0x5613efe3e0f3 <unknown>\n#24 0x5613efe4de7c <unknown>\n#25 0x7f135e294ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"fd4ec0e2a9fcca331b6307bd22e1dad3\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728781182641,
      "stop" : 1728781182641,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728781377925,
      "stop" : 1728781378701,
      "duration" : 776
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728781379336,
      "stop" : 1728781379397,
      "duration" : 61
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Used Resources Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "used_resources_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestUsedResourcesPage"
  }, {
    "name" : "host",
    "value" : "fv-az585-604"
  }, {
    "name" : "thread",
    "value" : "2289-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.used_resources_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 1,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 14,
        "unknown" : 0,
        "total" : 18
      },
      "items" : [ {
        "uid" : "d2bd4857647ee4ec",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/150//#testresult/d2bd4857647ee4ec",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5634592be02a <unknown>\n#1 0x563458fa45e0 <unknown>\n#2 0x563458f9c7eb <unknown>\n#3 0x563458f8ca09 <unknown>\n#4 0x563458f8e744 <unknown>\n#5 0x563458f8ccbd <unknown>\n#6 0x563458f8c52f <unknown>\n#7 0x563458f8c422 <unknown>\n#8 0x563458f8a493 <unknown>\n#9 0x563458f8aada <unknown>\n#10 0x563458fa6f41 <unknown>\n#11 0x5634590388c5 <unknown>\n#12 0x563459018b22 <unknown>\n#13 0x563459037d7d <unknown>\n#14 0x5634590188c3 <unknown>\n#15 0x563458fe66b3 <unknown>\n#16 0x563458fe768e <unknown>\n#17 0x563459288a2b <unknown>\n#18 0x56345928c9b1 <unknown>\n#19 0x563459275225 <unknown>\n#20 0x56345928d532 <unknown>\n#21 0x56345925a38f <unknown>\n#22 0x5634592acf28 <unknown>\n#23 0x5634592ad0f3 <unknown>\n#24 0x5634592bce7c <unknown>\n#25 0x7f50ad694ac3 <unknown>",
        "time" : {
          "start" : 1728694704961,
          "stop" : 1728694704961,
          "duration" : 0
        }
      }, {
        "uid" : "d6697baa290a750b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/d6697baa290a750b",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5583112e902a <unknown>\n#1 0x558310fcf5e0 <unknown>\n#2 0x558310fc77eb <unknown>\n#3 0x558310fb7a09 <unknown>\n#4 0x558310fb9744 <unknown>\n#5 0x558310fb7cbd <unknown>\n#6 0x558310fb752f <unknown>\n#7 0x558310fb7422 <unknown>\n#8 0x558310fb5493 <unknown>\n#9 0x558310fb5ada <unknown>\n#10 0x558310fd1f41 <unknown>\n#11 0x5583110638c5 <unknown>\n#12 0x558311043b22 <unknown>\n#13 0x558311062d7d <unknown>\n#14 0x5583110438c3 <unknown>\n#15 0x5583110116b3 <unknown>\n#16 0x55831101268e <unknown>\n#17 0x5583112b3a2b <unknown>\n#18 0x5583112b79b1 <unknown>\n#19 0x5583112a0225 <unknown>\n#20 0x5583112b8532 <unknown>\n#21 0x55831128538f <unknown>\n#22 0x5583112d7f28 <unknown>\n#23 0x5583112d80f3 <unknown>\n#24 0x5583112e7e7c <unknown>\n#25 0x7f3d30094ac3 <unknown>",
        "time" : {
          "start" : 1728608303675,
          "stop" : 1728608303675,
          "duration" : 0
        }
      }, {
        "uid" : "e48dd196a958dcd7",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/e48dd196a958dcd7",
        "status" : "passed",
        "time" : {
          "start" : 1728558066136,
          "stop" : 1728558256602,
          "duration" : 190466
        }
      }, {
        "uid" : "c5bc093f50363d61",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/c5bc093f50363d61",
        "status" : "passed",
        "time" : {
          "start" : 1728554962801,
          "stop" : 1728555156310,
          "duration" : 193509
        }
      }, {
        "uid" : "21bb13d87970cf69",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/21bb13d87970cf69",
        "status" : "passed",
        "time" : {
          "start" : 1728522890063,
          "stop" : 1728522891203,
          "duration" : 1140
        }
      }, {
        "uid" : "4f574a98fd9f2365",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/4f574a98fd9f2365",
        "status" : "passed",
        "time" : {
          "start" : 1728471165688,
          "stop" : 1728471166800,
          "duration" : 1112
        }
      }, {
        "uid" : "585905b0d8a72c92",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/585905b0d8a72c92",
        "status" : "passed",
        "time" : {
          "start" : 1728468633799,
          "stop" : 1728468635360,
          "duration" : 1561
        }
      }, {
        "uid" : "ecfb8704e01115fa",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/ecfb8704e01115fa",
        "status" : "passed",
        "time" : {
          "start" : 1728436615174,
          "stop" : 1728436616296,
          "duration" : 1122
        }
      }, {
        "uid" : "f6ddc2eeca492607",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/f6ddc2eeca492607",
        "status" : "passed",
        "time" : {
          "start" : 1728378582352,
          "stop" : 1728378583539,
          "duration" : 1187
        }
      }, {
        "uid" : "8dab317026f4e8a9",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/8dab317026f4e8a9",
        "status" : "passed",
        "time" : {
          "start" : 1728375905083,
          "stop" : 1728375906179,
          "duration" : 1096
        }
      }, {
        "uid" : "e955a82116a1510a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/e955a82116a1510a",
        "status" : "passed",
        "time" : {
          "start" : 1728375535348,
          "stop" : 1728375536770,
          "duration" : 1422
        }
      }, {
        "uid" : "4734bd19dae5afba",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/4734bd19dae5afba",
        "status" : "passed",
        "time" : {
          "start" : 1728349607241,
          "stop" : 1728349608468,
          "duration" : 1227
        }
      }, {
        "uid" : "a11fc39655f11df9",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/a11fc39655f11df9",
        "status" : "passed",
        "time" : {
          "start" : 1728293693337,
          "stop" : 1728293694469,
          "duration" : 1132
        }
      }, {
        "uid" : "ccdc8129074659e0",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/ccdc8129074659e0",
        "status" : "passed",
        "time" : {
          "start" : 1728290759108,
          "stop" : 1728290777400,
          "duration" : 18292
        }
      }, {
        "uid" : "f781d2e211a6f4ba",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/f781d2e211a6f4ba",
        "status" : "passed",
        "time" : {
          "start" : 1728264401035,
          "stop" : 1728264405534,
          "duration" : 4499
        }
      }, {
        "uid" : "747872fc9432d4f3",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/747872fc9432d4f3",
        "status" : "failed",
        "statusDetails" : "AssertionError: Links in the sections lead to incorrect pages after clicking\nassert False\n +  where False = all(<generator object TestUsedResourcesPage.TestUsedResourcesPageLinks.test_ur_03_02_verify_links_lead_to_the_correct_pages.<locals>.<genexpr> at 0x7f2fe9216740>)",
        "time" : {
          "start" : 1728227607885,
          "stop" : 1728227659352,
          "duration" : 51467
        }
      }, {
        "uid" : "b5ad11f5b21443ae",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/b5ad11f5b21443ae",
        "status" : "passed",
        "time" : {
          "start" : 1728222667353,
          "stop" : 1728222670104,
          "duration" : 2751
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "27c2c12f61744d9.json",
  "parameterValues" : [ ]
}